torch/csrc/lazy/python/init.cpp

```diff
@@ -10,10 +10,10 @@
 #include <torch/csrc/lazy/backend/backend_interface.h>
 #include <torch/csrc/lazy/core/ir_dump_util.h>
 #include <torch/csrc/lazy/core/internal_ops/ltc_ops.h>
-#include <torch/csrc/lazy/ts_backend/ops/device_data.h>
```

Comment:
@antoniojkim This change is the right thing to do, but it also broke internal CI since there is a usage of TSComputation in one of the python bindings below.

The fix is to stop compiling that ts binding in fbcode builds.
https://github.com/pytorch/pytorch/pull/76337

I can either land 76337 and you rebase, or you can just copy the fix into your PR which would probably save time

Reply:
cc @shunting314